### ISSUE TITLE
Not working.

### ISSUE THREAD

**Issue by greenhell1337** [luv4code-lc/Intelli_Idea_Ultimate_2021.3#3]
I do everything according to the instructions, but he does not see the file on this path. Precisely, that he even ./idea.sh he doesn't want to open it.
![image](https://user-images.githubusercontent.com/95542529/187021862-02d1db25-323b-4dde-9177-49a593a0eead.png)

**Comment by luv4code-lc** [comment]
Firstly you have to download the  fineagent jar file

Go  to installation directory

Open idea64.vmoptions file using vim or some editing software


Just copy and paste
-javaagaent:/ <jar file location>

**Comment by greenhell1337** [comment]
![image](https://user-images.githubusercontent.com/95542529/187022541-b15ae414-3a20-4b00-a465-7f44f76c09fd.png)
![image](https://user-images.githubusercontent.com/95542529/187022556-891f396f-e5c0-482d-9b08-70dbf0993e84.png)
Don't think I'm some kind of fool. I know I made a mistake, but I installed fineagent.

**Comment by luv4code-lc** [comment]
I have installed more than 5 times in ubuntu it works well for me

**Comment by luv4code-lc** [comment]
Ok I will show you with screenshots just wait for sometime

**Comment by greenhell1337** [comment]
I do not exclude the possibility that I made a mistake in something. I'll be waiting.

**Comment by greenhell1337** [comment]
it's amazing. I don't understand why, but the program just doesn't see the desired jar file along this path. I'm sure for sure that you have nothing to do with it, something is wrong with my settings, but I don't understand where I made a mistake. after I reinstalled the system for the first time, everything worked, but after closing the program and trying to open it again, the same error occurred.
